README.md

```diff
@@ -25,8 +25,8 @@ buildscript {
 }
 
 dependencies {
-  annotationProcessor 'com.uber:rave-compiler:2.0.0'
-  compile 'com.uber:rave:2.0.0'
+    annotationProcessor 'com.uber:rave-compiler:2.0.0'
```

Comment:
We use 2 space indents, please revert this

Reply:
But Android Studio - Auto Format give 4 indents

Reply:
the default maybe, but each project can have their own. Our style is 2

Reply:
ok